installation/sql/postgresql/base.sql

```diff
@@ -347,6 +347,7 @@ INSERT INTO "#__extensions" ("package_id", "name", "type", "element", "folder",
 (0, 'plg_schemaorg_organization', 'plugin', 'organization', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 6, 0),
 (0, 'plg_schemaorg_person', 'plugin', 'person', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 7, 0),
 (0, 'plg_schemaorg_recipe', 'plugin', 'recipe', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 8, 0),
+(0, 'plg_schemaorg_product', 'plugin', 'product', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 9, 0),
```

Comment:
```suggestion
(0, 'plg_schemaorg_product', 'plugin', 'product', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 8, 0),
(0, 'plg_schemaorg_recipe', 'plugin', 'recipe', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 9, 0),
```
Alpha ordering the plugins of the same type (folder).